qa/workunits/mon/rbd_snaps_ops.sh

```diff
@@ -19,7 +19,7 @@ expect()
 }
 
 ceph osd pool delete test test --yes-i-really-really-mean-it || true
-expect 'ceph osd pool create test 256 256' 0
+expect 'ceph osd pool create test 8' 0
 expect 'rbd --pool=test pool init' 0
```

Comment:
We also have to remove this line to make it work. As since some time `rbd pool init` have been validating the pool by creating a self-managed snapshot.

Also, at the end of the test, `rbd --pool test-bar snap rm image@snapshot` was expected to return 95 (when running on a copied pool (i.e. when snap_id has not been actually reserved), but currently it returns 0 (the snap remove request just proceeds when `release_snap_id` returns `-ENOENT`).

Reply:
Wow -- so this has been broken since Nautilus I guess. I replaced all the "rbd pool init" calls with just "ceph osd pool application enable" calls to prevent librbd from creating self-managed snapshots in the pools. That seems to have addressed the issues.